chucks-pick3-web/style-sheet.css

```diff
@@ -0,0 +1,133 @@
+.main-container {
```

Comment:
Please create a `assets/css` directory and move this file into the css directory.  Be sure to update the home.html and the home-template.html files.

Reply:
Completed directory changes and updated both html pages.